pkgs/development/python-modules/pytest-factoryboy/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pytestCheckHook
+, pytest
+, inflection
+, factory_boy
+, pytestcache
+, pytestcov
+, pytestpep8
+, mock
```

Comment:
My personal preference is to separate ``checkInputs`` from ``propagatedBuildInputs`` for python packages in this list, but either way is fine.